Sources/MapboxNavigation/NavigationCameraStateTransition.swift

```diff
@@ -102,6 +107,16 @@ public class NavigationCameraStateTransition: CameraStateTransition {
     }
     
     public func updateForFollowing(_ cameraOptions: CameraOptions) {
+        // Check whether the original and target locations and headings are larger than a certain threshold when current camera state is following.
+        guard let mapView = mapView,
+              let center = cameraOptions.center,
+              let zoom = cameraOptions.zoom,
+              let bearing = cameraOptions.bearing else { return }
+
+        let centerUpdateThreshould = baseCenterUpdateThreshould * pow(2, 22.0 - Double(zoom))
+        guard mapView.cameraState.center.distance(to: center) > centerUpdateThreshould,
+              abs(mapView.cameraState.bearing - bearing) > bearingUpdateThreshold else { return }
```

Comment:
Does this expression need to account for the possibility that  –1 can signify an unset `CLLocationDirection` value?

Reply:
I'll add the `=` for it.